### PR TITLE
Remove StatsSnapshot object

### PR DESCRIPTION
**Pull request by justinlin-linkedin**
Clean up the old code

### CODE REVIEW

File: ambry-quota/src/main/java/com/github/ambry/quota/storage/MySqlStorageUsageRefresher.java [hunk 1]
```diff
@@ -359,6 +366,32 @@ static long secondsToNextMonthTick(String currentMonthValue, long offsetInSecond
...
   }
 
+  /**
+   * Convert an {@link AggregatedAccountStorageStats} to a map from account id to container id to storage usage.
+   * The account id and container id are keys of the outer map and the inner map, they are both in string format.
+   * @param aggregatedAccountStorageStats The {@link AggregatedAccountStorageStats}.
+   * @param usePhysicalStorageUsage True to use physical storage, false to use logical storage usage.
+   * @return A map from account id to container id to storage usage.
+   */
+  static Map<String, Map<String, Long>> convertAggregatedAccountStorageStatsToMap(
```
Comment (SophieGuo410) [inline ambry-quota/src/main/java/com/github/ambry/quota/storage/MySqlStorageUsageRefresher.java:376]: Why did you move this method out of StorageStatsUtil? Seems that StorageStatsUtil.java is empty now and we can remove it?

File: ambry-clustermap/src/main/java/com/github/ambry/clustermap/MySqlReportAggregatorTask.java [hunk 2]
```diff
@@ -175,8 +172,8 @@ public TaskResult run() {
       return new TaskResult(TaskResult.Status.FAILED, "Exception thrown");
     } finally {
       if (clusterMapConfig.clustermapEnableContainerDeletionAggregation && callback != null
```
Comment (SophieGuo410) [inline ambry-clustermap/src/main/java/com/github/ambry/clustermap/MySqlReportAggregatorTask.java:174]: I'll remove this part of logic after your changes since it won't be used any more cause we leverage orca job to do it.

File: ambry-quota/src/main/java/com/github/ambry/quota/storage/MySqlStorageUsageRefresher.java (hunk 1 above)
Comment (justinlin-linkedin) [reply ambry-quota/src/main/java/com/github/ambry/quota/storage/MySqlStorageUsageRefresher.java:376]: removed

File: ambry-clustermap/src/main/java/com/github/ambry/clustermap/MySqlReportAggregatorTask.java (hunk 2 above)
Comment (justinlin-linkedin) [reply ambry-clustermap/src/main/java/com/github/ambry/clustermap/MySqlReportAggregatorTask.java:174]: sure, let's do this in next PR.

**Review by SophieGuo410 — APPROVED** [review]

**Review by cgtz — APPROVED** [review]
